packages/console/src/cloud/pages/Main/index.tsx

```diff
@@ -1,5 +1,77 @@
+import { useLogto } from '@logto/react';
+import type { TenantInfo } from '@logto/schemas';
+import { trySafe } from '@silverhand/essentials';
+import { useCallback, useContext, useEffect } from 'react';
+import { useHref, useNavigate } from 'react-router-dom';
+
+import { useCloudApi } from '@/cloud/hooks/use-cloud-api';
+import AppLoading from '@/components/AppLoading';
+import { getUserTenantId } from '@/consts/tenants';
+import { TenantsContext } from '@/contexts/TenantsProvider';
+
 const Main = () => {
-  return <div>Main</div>;
+  const { isAuthenticated, isLoading, signIn, getAccessToken } = useLogto();
+  const api = useCloudApi();
+  const {
+    tenants: { data, isSettle },
+    setTenants,
+  } = useContext(TenantsContext);
+  const navigate = useNavigate();
+  const href = useHref(getUserTenantId() + '/callback');
+
+  const loadTenants = useCallback(async () => {
+    const data = await api.get('/api/tenants').json<TenantInfo[]>();
```

Comment:
Why don't we use `useSWR`?
Maybe we can use the `SWR` in the console and manage all requests in the same `SWR` logic.
FYI: https://github.com/logto-io/logto/pull/3236
cc @simeng-li

Reply:
- `useSWR` is good to try, i'll do it in the next PR since there's some change on data fetching
- this is inside the cloud app, i'm not sure if it's good to share the same `useSWR`.